Updates Doc for Source*Id (#2284)
FusionAuth/fusionauth-site: site/docs/v1/tech/apis/tenants.adoc

The following APIs are provided to manage Tenants.

* <<Create a Tenant>>

> * <<Create a New Tenant From an Existing Tenant>>

Reviewer: Same as the application comment, use sentence capitalization here. 

`Create a Tenant from an existing Tenant`

include::docs/v1/tech/apis/_tenant-response-body.adoc[]

== Create a New Tenant From an Existing Tenant

> This API is used to create a new Tenant from an existing Tenant.

Reviewer 2: I could go either way on whether we call this out here, but I think it should be consistent.

```suggestion
This API is used to create a new Tenant from an existing Tenant.

Specifying an Id on the URI will instruct FusionAuth to use that Id when creating the Tenant. Otherwise, FusionAuth will generate an Id for the Tenant.
```

Author: I dont think that we need it

=== Request

[.api-authentication]

> link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Create a Tenant with a randomly generated Id

Reviewer 2: ```suggestion
link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Create a new Tenant from an existing Tenant with a generated Id
```

[.endpoint]
.URI
--
[method]#POST# [uri]#/api/tenant#
--

[.api-authentication]

> link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Create a Tenant with the provided unique Id

Reviewer 2: ```suggestion
link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Create a new Tenant from an existing Tenant with the given Id
```

Author: We use `provided unique` interchangeably with `given` in our docs.  Is there a preference here?

Reviewer 2: I was only comparing the changes for this PR. I doubt whether anyone would notice that these are phrased differently if they weren't seeing them all side by side.

[.endpoint]
.URI
--
[method]#POST# [uri]#/api/tenant/`\{tenantId\}`#
--

==== Request Parameters

[.api]
[field]#tenantId# [type]#[UUID]# [optional]#Optional# [default]#defaults to secure random UUID#::
The Id to use for the new Tenant. If not specified a secure random UUID will be generated.

==== Request Body

[.api]

> [field]#sourceTenantId# [type]#[UUID]# [required]#Required# [since]#Available since 1.14.0#::

Reviewer 2: Move the `Available since` just under the section heading?

> The Id of an existing Tenant to make a copy of. A unique [field]#tenant.name# is required. All other values will be copied from the source Tenant to the new Tenant.

Reviewer 2: ```suggestion
The Id of an existing Tenant from which a copy will be made. A unique [field]#tenant.name# is required. All other values will be copied from the source Tenant to the new Tenant.
```

[field]#tenant.name# [type]#[String]# [required]#Required#::
The unique name of the Tenant.

.Example request JSON
[source,json]

> {
> 	"sourceTenantId": "32306536-3036-6431-3865-646430303332",
> 	"tenant": {
> 		"name": "Playtronics Co. - copied"
> 	}
> }

Reviewer 2: ```suggestion
{
  "sourceTenantId": "32306536-3036-6431-3865-646430303332",
  "tenant": {
    "name": "Playtronics Co. - copied"
  }
}
```
